libcxx/test/std/utilities/template.bitset/bitset.members/to_ullong.pass.cpp

```diff
@@ -14,35 +14,40 @@
 #include <climits>
 #include <cassert>
 
+#ifndef TEST_HAS_NO_EXCEPTIONS
+#  include <stdexcept>
+#endif
```

Comment:
```suggestion
#include <stdexcept>
```

You can always include this, that's okay.